CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.10.0 (Released 2025-03-04)
+
+BREAKING CHANGES
+
+- transfers: switch to newer `/accounts/{accountID}`-prefixed endpoints
+- transfers: update transfer methods to accept an accountID parameter
+- transfers: add cancellation API endpoints and client methods
```

Comment:
this isn't a breaking change, would put this under ADDITIONS

Reply:
it wouldn't? i was thinking it was a breaking change since if they were using this new version, calls to existing client methods would error because they now have to pass in accountID as param.. unless I'm misinterpreting what "breaking change" means

Reply:
breaking change = new behavior in the API?

Reply:
i'm referring to the new cancellation methods -- those are only additive

Reply:
got it- placed in a separate section `ADDITIONS`